packages/data-store/src/entities/xstate/StateEntity.ts

```diff
@@ -0,0 +1,47 @@
+import { BaseEntity, Column, CreateDateColumn, Entity, PrimaryGeneratedColumn, UpdateDateColumn } from 'typeorm'
+import { NonPersistedXStateStoreEvent } from '../../types'
+
+@Entity('StateEntity')
+export class StateEntity extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid', { name: 'id' })
+  id!: string
```

Comment:
This should not be generated, as every Xstate instance already has a unique id AFAIK

Reply:
Okay apparently the generated value is not unique. So we need something like this to ensure uniqueness. Will take care of that

Reply:
Currently we have an id that can be passed in and is expected to be globally unique. Otherwise it gets generated.
There is also a sessionId, that takes the "unique" value from Xstate